main_test.go

```diff
@@ -102,6 +102,30 @@ func TestRunDaemon(t *testing.T) {
 	assert.Contains(t, buf.String(), "forced wake-up from sleep", "daemon was not forced from sleep")
 }
 
+// Tests that the client will boot with an error message in the case of an invalid server certificate.
+func TestInvalidServerCertificateBoot(t *testing.T) {
+	tdir, err := ioutil.TempDir("", "invalidcert-test")
+	require.Nil(t, err)
+
+	fmt.Printf("%v", log.Log)
+	logBuf := bytes.NewBuffer(nil)
+	log.SetLevel(log.DebugLevel)
```

Comment:
I don't think we need to set the level to Debug for this test

Reply:
By the way, here for instance you are not defering a call to restore the original level, meaning "you are not cleaning up after yourself". This might trigger errors like the one you showed me, where the order of the test cases alter the results.

Reply:
You are absolutely right  of course. Sloppy. I'm sry.